.ci/deps.elm-platform-packages.ps1

```diff
@@ -0,0 +1,12 @@
+function Install-Elm-Format {
```

Comment:
used atm, not sure why I didnt get this choco package working - it is unnecessary because elm is also installed via npm
ftr
```
function Install-Elm-Format {
    elm-compiler --version
    wget https://github.com/avh4/elm-format/releases/download/0.8.1/elm-format-0.8.1-win-i386.zip
    7z e elm-format-0.8.1-win-i386.zip
    mv elm-format.exe ./node_modules/.bin
    touch elm-package.json
This conversation was marked as resolved by jayvdb
    # For elm 0.19, touch elm.json instead.
}

function Invoke-ExtraInstallation {
    Install-Elm-Format
}
```